test/e2e/integration/003-multi-block-selection.js

```diff
@@ -0,0 +1,58 @@
+describe( 'Multi-block selection', () => {
+	before( () => {
+		cy.newPost();
+	} );
+
+	it( 'Should select/unselect multiple blocks', () => {
+		const lastBlockSelector = '.editor-visual-editor__block-edit:last [contenteditable="true"]:first';
+		const firstBlockContainerSelector = '.editor-visual-editor__block:first';
+		const lastBlockContainerSelector = '.editor-visual-editor__block:last';
+		const multiSelectedCssClass = 'is-multi-selected';
```

Comment:
Raised before on other PRs. We should consolidate these selectors in shared variables or use data-test attributes